demo 2/src/main/java/com/example/demo/service/ItemService.java

```diff
@@ -0,0 +1,166 @@
+package com.example.demo.service;
+
+import com.example.demo.constants.ResultInfoConstants;
+import com.example.demo.entity.Item;
+import com.example.demo.exceptions.AccountNotFoundException;
+import com.example.demo.repository.table.*;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@RequiredArgsConstructor
+@Service
+@Slf4j
+public class ItemService {
+
+    private final ItemRepository itemRepository;
+
+    private final AccountRepository accountRepository;
+
+    private final FolderRepository folderRepository;
+
+    public List<Item> getMap(Long phone) {
+        AccountTable accountTable = accountRepository.getUser(phone);
+        if (accountTable == null) {
+            log.warn("Invalid phone number");
+            throw new AccountNotFoundException(ResultInfoConstants.ACCOUNT_NOT_FOUND);
+        }
+        if (!accountTable.isLogin()) {
+            log.warn("Need to login");
+            throw new AccountNotFoundException(ResultInfoConstants.NEED_TO_LOGIN);
+        }
+        return itemRepository.findByPhone(phone).stream().map(ItemTable::toItem).collect(Collectors.toList());
+    }
+
+    public String insert(Item item, Long phone) {
+        AccountTable accountTable = accountRepository.getUser(phone);
+        if (accountTable == null) {
+            log.warn("Invalid phone number");
+            throw new AccountNotFoundException(ResultInfoConstants.ACCOUNT_NOT_FOUND);
+        }
+        if (!accountTable.isLogin()) {
+            log.warn("Need to login");
+            throw new AccountNotFoundException(ResultInfoConstants.NEED_TO_LOGIN);
+        }
+       /*UrlValidator urlValidator = new UrlValidator(UrlValidator.ALLOW_LOCAL_URLS);
+        UrlValidator urlValidator = new URLValidator();
+          if (urlValidator.isValid()){
+            log.warn("Enter valid url");
+            throw new DuplicateKeyException(ResultInfoConstants.ENTER_VALID_URL);
+        }*/
+
+        Long folderId = item.getFolder();
+        FolderTable folderTable = folderRepository.getFolderByIdAndPhone(folderId, phone);
+        if (folderTable == null) {
+            log.warn("Invalid folder id");
+            throw new AccountNotFoundException(ResultInfoConstants.FOLDER_NOT_FOUND);
+        }
+
+        ItemTable itemTable = item.toItemTable(folderTable.getId());
```

Comment:
Should encrypt username and password.
And return decrypted username and password in response for GET Apis